src/containers/feedOfClaim/index.tsx

```diff
@@ -173,11 +173,10 @@ const FeedClaim: React.FC<IHomeProps> = ({ toggleTheme, isDarkMode }) => {
   }
 
   // Handles graph navigation
-  const handleschema = async (nodeUri: string) => {
-    const domain = nodeUri.replace(/^https?:\/\//, '').replace(/\/$/, '')
+  const handleNavigationToGraph = async (claimId: string) => {
     navigate({
       pathname: '/search',
-      search: `?query=${domain}`
+      search: `?query=${claimId}`
```

Comment:
actually it should work with the url - we should not care what it is

it should not be doing this tho `replace(/^https?:\/\//, '')`

i need to walk thru and understand what people are thinking here we may need more refactor and semantic code, this doesn't quite make sense to me. 

Can you give sample searches for which it fails, and what the behavior is?

Reply:
![image](https://github.com/Whats-Cookin/trust_claim/assets/106309885/1fce96e6-2173-48cf-b7b6-6635b0a6778f)
back end doesn't allow searching with urls currently so frontend deal with them (linkedin urls and other urls) separately.
i was going to ask you to do some refactoring to the front end as well as the back end since the code is messy and hard to maintain but i wanted to fix the issue as soon as possible first.

Reply:
please don't guess!   it has nothing to do with linkedin urls!